modules/EnsEMBL/Draw/GlyphSet_transcript_new.pm

```diff
@@ -254,6 +254,10 @@ sub get_gene_connections {
   return unless $ma;
   
   my $qy_member = $ma->fetch_by_stable_id($gene->stable_id);
+  my $species_prod_name = $gene->adaptor->db->get_MetaContainer->get_production_name();
+  my $genome_db = $compara_db->get_GenomeDBAdaptor->fetch_by_name_assembly($species_prod_name);
+  
+  my $qy_member = $ma->fetch_by_stable_id_GenomeDB($gene->stable_id, $genome_db);
```

Comment:
I don't think you need the `my` on this line. It's already declared on 256, so having it here too will generate a warning

Reply:
Well that kine should have been replaced with new lines. Updated.